gui/gui.py

```diff
@@ -266,6 +267,7 @@ def run_simulation(name):
 
     window = setup_gui()
     current_row = 0
+    rs = RadioSession(False)
```

Comment:
I didn't work on this code, but I don't know if the code would still run if it was moved to section 3. rs is still referred to in a later part in sec 2, line 300 ( rs.setup_basestation() is dependent on 'event' which is defined in sec 2).

Reply:
I didn't work on this file but I think it would have to stay in section 2 since rs is referenced later in section 2 with  rs.setup_basestation(), which is called depending on what the event is. And event is defined in section 2.